org.gaphor.Gaphor.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: org.gaphor.Gaphor
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+rename-icon: gaphor
```

Comment:
The rename should also happen upstream.

Reply:
@barthalion Can you please help me understand how to rename the icon upstream? I'm not sure I understand this. I was following the [Flatpak dictionary example](https://github.com/flathub/org.gnome.Dictionary/blob/master/org.gnome.Dictionary.json)

Reply:
In general the filename should match the app-id you're using, according to the freedesktop icon spec. It's not a blocker and can be handled later though.